### ISSUE TITLE
The action accuracy after fine-tuning does not reach ~0.95

### ISSUE THREAD

**Issue by HanwenWangBUPT** [openvla/openvla#74]
Thank you for your outstanding work. I am currently fine-tuning OpenVLA based on LoRA using a simple grasping action dataset that I collected myself(~100 Trajectories). However, during training, I am unable to achieve the action accuracy (~0.95) that you mentioned. Although my training loss has converged, I would like to understand what potential issues might be causing this.

**Fine-tuning data example**
![data_image](https://github.com/user-attachments/assets/ba0a1676-3bcc-48c6-820d-ad81da5e37b5)

**Fine-tuning Parameters**
` batch_size: int = 1 `                                  
` max_steps: int = 200_000 `                                     
` save_steps: int = 5000 `                                        
` earning_rate: float = 1e-5 `                                   
` grad_accumulation_steps: int = 16 `                            
`  image_aug: bool = True `                                        
` shuffle_buffer_size: int = 100_000 ` 

**Fine-tuning status**
![6e958a8eefe54cb4b2c112a0ddee3b5](https://github.com/user-attachments/assets/c8bc1cde-94ae-4a43-876a-868e4c50ac3d)

**Comment by kpertsch** [comment]
Hi!
Thanks for trying out OpenVLA! Your robot setup looks great!
I am a bit surprised that the model doesn't fit the data. One thing that's a bit weird is that your accuracy curve has these very discrete values (it looks like it's computed over 4 action samples only?) and seems very noisy. Maybe one thing to try would be to increase the batch size of your run? it looks like you may have a bug in how the batch size of your run is computed?

**Comment by kpertsch** [comment]
Will close for now, feel free to reopen!

**Comment by HanwenWangBUPT** [comment]
Thank you for your response. I did encounter some bugs with the `batch_size` settings, but I've resolved them now.

**Comment by YEOMJINSEOP** [comment]
Hi, @HanwenWangBUPT  i'm also experiencing noise in experiment results. Could you share how you handle batch_size setting? Thanks!

**Comment by HanwenWangBUPT** [comment]
> Hi, @HanwenWangBUPT i'm also experiencing noise in experiment results. Could you share how you handle batch_size setting? Thanks! 
If your device's GPU memory cannot support a large `batch_size`, you need to ensure that `grad_accumulation_steps` set to 16, also you need to ensure that your code successfully passes the parameter. Hope this can help you.
